src/base/pages/Landing.vue

```diff
@@ -281,88 +267,394 @@ export default {
+      return require(`@/base/pages/images/${dirName}/${fileName}-${width}${ext}`)
+    },
   },
 }
 </script>
 
 <style scoped lang="stylus">
 @import '~variables'
+$box-shadow-color = 84, 84, 84
+
+>>>
+  --h1-fontsize 1.75rem // 28px
+  --h1-letterspacing -0.005em
+  --h2-fontsize 1.375rem // 22px
+  --paragraph-fontsize 1.0625rem // 17px
+  --landing-padding-top 30px
+  --landing-padding-x 20px
+  --img-border-radius 4px
+  --app-screenshots-padding 25px 0 25px var(--landing-padding-x)
+  --app-screenshots-box-sdw 0 3px 22px -3px rgba(0, 0, 0, 0.2)
+  --app-screenshots-devices-bg-color #FFF5D9
+  --app-screenshots-outer-wrapper-width 150%
+  --app-screenshots-browser-top-bar-position absolute
+  --app-screenshots-browser-top-bar-height 18px
+  --app-screenshots-browser-top-bar-border-radius 10px
+  --app-screenshots-browser-traffic-light-size 6px
+  --app-screenshots-browser-traffic-light-margin 3px
+  --app-screenshots-browser-width 132%
+  --app-screenshots-browser-img-bg-sizing contain
+  --app-screenshots-browser-img-position-y var(--app-screenshots-browser-top-bar-height)
+  --app-screenshots-browser-img-offset 25px
+  --app-screenshots-phone-position relative
+  --app-screenshots-phone-width 43%
+  --app-screenshots-phone-right unset
+  --app-screenshots-phone-bottom unset
+
+  @media (min-width: 410px)
+    --h1-fontsize 2.375rem // 38px
+
+  @media (min-width: 500px)
+    --landing-padding-top 45px
+    --landing-padding-x 40px
+    --app-screenshots-browser-top-bar-height 24px
+    --app-screenshots-browser-top-bar-border-radius 13px
+    --app-screenshots-browser-traffic-light-size 8px
+    --app-screenshots-browser-traffic-light-margin 4px
+    --app-screenshots-browser-img-offset 34px
 
-.landing
+  @media (min-width: 600px)
+    --h1-fontsize 2.75rem // 44px
+    --h1-letterspacing -0.025em
+    --h2-fontsize 1.875rem // 30px
+    --paragraph-fontsize 1.25rem // 20px
+
+  @media (min-width: 700px)
+    --app-screenshots-padding 55px 120px 65px var(--landing-padding-x)
+    --app-screenshots-outer-wrapper-width 100%
+    --app-screenshots-browser-top-bar-position relative
+    --app-screenshots-browser-top-bar-height 34px
+    --app-screenshots-browser-top-bar-border-radius 15px
+    --app-screenshots-browser-traffic-light-size 10px
+    --app-screenshots-browser-traffic-light-margin 6px
+    --app-screenshots-browser-width 100%
+    --app-screenshots-browser-img-bg-sizing cover
+    --app-screenshots-browser-img-position-y 50%
+    --app-screenshots-browser-img-offset 0px
+    --app-screenshots-phone-position absolute
+    --app-screenshots-phone-width 32.5%
+    --app-screenshots-phone-right -105px
+    --app-screenshots-phone-bottom -27px
+
+  @media (min-width: 850px)
+    --h1-fontsize 3.125rem // 50px
+    --app-screenshots-phone-width 31.023%
 
+  @media (min-width: 1050px)
+    --landing-padding-top 60px
+    --landing-padding-x 100px
+    --app-screenshots-padding 55px 120px 65px 50px
```

Comment:
Initially I was a bit overwhelmed by all the vars, but I think I get it now! It's to collect all the variables that change at different screen widths, which then get used in the various other bits of style (to avoid n*m @media query and css selector combinations).

Still a bit overwhelming, but as it's a one-off page, and needs to be the :sparkles: _shiniest_ :sparkles:  page of all, seems good! :+1:

Reply:
Haha, yes, shiny sometimes means some degree of complexity 😆 
And also yes, not using css-vars would have meant to use dozens of **@media** queries making the css even harder to read IMO.